packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/local/translations_shared_data_source.dart

```diff
@@ -0,0 +1,24 @@
+{{> licence.dart }}
+
+import 'dart:convert';
+
+import '../../models/i18n/i18n_models.dart';
+import 'shared_preferences_instance.dart';
+import 'translations_local_data_source.dart';
+
+class TranslationsSharedDataSource extends TranslationsDataSource {
```

Comment:
Please remove the local storage providers. Let's use the in-memory storage ot keep the solution as minimal as possible